app/controllers/api/v1/carts_controller.rb

```diff
@@ -33,6 +33,61 @@ def show
             end
 
 
+            def add
+
+                @cart = Cart.find(params[:id])
+                puts(@cart.item_numbers)
+                
+                @cart.item_numbers.push(params[:item_number])
+                # @cart.item_numbers_will_change!
+                # @cart.save!
+
+                puts(@cart.inspect)
+                if @cart.save!
+                    render json: @cart, status: :created
+                else
+                    render json: @cart.errors, status: :unprocessable_entity
+                end
+
+            end
+
+
+            def itemfetch
+                @cart = Cart.find(params[:id])
+
+                #assumes coming in as an array
+                @itemlot = @cart.item_numbers
+
+                @arrayofitems = ["Array of Cart Items"]
```

Comment:
Eventually we need to change this so this so we can have a 0 index that is not  "Array of Cart Items".